train-aws.gemspec

```diff
@@ -117,7 +117,7 @@ Gem::Specification.new do |spec|
   # spec.add_dependency "aws-sdk-glue", "~> 1.0"
   # spec.add_dependency "aws-sdk-greengrass", "~> 1.0"
   # spec.add_dependency "aws-sdk-groundstation", "~> 1.0"
-  # spec.add_dependency "aws-sdk-guardduty", "~> 1.0"
+  spec.add_dependency "aws-sdk-guardduty", "~> 1.0"
```

Comment:
I see the newest version of guardduty is at `1.31`. Should we consider updating this to the latest?

Reply:
Seems worthwhile to me